config/samples/all_enabled_openshift.yaml

```diff
@@ -240,99 +214,29 @@ spec:
       - openshift-service-ca-operator
       - openshift-user-workload-monitoring
 
-    kube-state-metrics:
-      image:
-        repository: public.ecr.aws/sumologic/kube-state-metrics
-        tag: v1.9.7-ubi
-
     prometheus-node-exporter:
-      image:
-        repository: public.ecr.aws/sumologic/node-exporter
-        tag: v1.0.1-ubi
       service:
         port: 9200
         targetPort: 9200
 
-    prometheus:
-      prometheusSpec:
-        image:
-          repository: public.ecr.aws/sumologic/prometheus
-          tag: v2.22.1-ubi
-
-        thanos:
-          image: public.ecr.aws/sumologic/thanos:v0.10.0-ubi
-
   ## Configure optional OpenTelemetry Collector in Agent mode
   otelagent:
-    enabled: false
-    daemonset:
-      image:
-        repository: "public.ecr.aws/sumologic/opentelemetry-collector"
-        tag: "0.22.0-sumo-ubi"
-
-  ## Configure otelcol
-  otelcol:
-    deployment:
-      image:
-        repository: "public.ecr.aws/sumologic/opentelemetry-collector"
-        tag: "0.22.0-sumo-ubi"
+    enabled: true
 
   ## Configure telegraf-operator
   ## This is an experimental configuration and shouldn't be used in production environment
```

Comment:
Why we say this is an experimental?

Reply:
It is just copied from values.yaml, see: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v2.1/deploy/helm/sumologic/values.yaml#L2303-L2305